File: 3331-MinimumOperationsToExceedThresholdValueI/3331-MinimumOperationsToExceedThresholdValueI.java
// Last updated: 31/07/2025, 21:05:38
class Solution {
    public int minOperations(int[] nums, int k) {
        int l=nums.length;
        Arrays.sort(nums);
        int c=0;
        for(int i=0;i<l;i++)
        {
            if(nums[i]>=k)
            {
                break;
            }
            else
            {
                c++;
            }
        }
        return c;
        }
    
    }

File: 3331-MinimumOperationsToExceedThresholdValueI/3331-MinimumOperationsToExceedThresholdValueI.py
# Last updated: 31/07/2025, 21:05:34
class Solution:
    def minOperations(self, nums: List[int], k: int) -> int:
        nums.sort()
        c=0
        for i in range(len(nums)):
            if nums[i]>=k:
                break
            else:
                c+=1;
        return c            
        